packages/core/src/payment-provider-customer/payment-provider-customer-actions-creator.ts

```diff
@@ -0,0 +1,20 @@
+import { createAction } from '@bigcommerce/data-store';
+import { Observable, of } from 'rxjs';
+
+import {
+    PaymentProviderCustomerAction,
+    PaymentProviderCustomerType,
+} from './payment-provider-customer-actions';
+
+export default class PaymentProviderCustomerActionCreator {
+    updatePaymentProviderCustomer<T = object>(
+        providerCustomerData: T,
+    ): Observable<PaymentProviderCustomerAction> {
+        return of(
+            createAction(
+                PaymentProviderCustomerType.PaymentProviderCustomerCollected,
```

Comment:
```suggestion
                PaymentProviderCustomerType.UpdatePaymentProviderCustomer,
```

Reply:
Thanks, it looks much meaningful. Updated.